client/src/component/User.js

```diff
@@ -1,10 +1,30 @@
+import React, { useState } from "react";
+import "./User.css";
+
+
 export function User() {
 
+	const [name, setName] = useState("");
+
+	const handleInputChange = (event) => {
+		setName(event.target.value);
+	};
+
+	// const handleSearch = () => {
+	//   // Perform the search logic here using the "name" value
+	//   // You can make an API call or search the data locally
+
+	//   // Example: Logging the name value
+	//   console.log(`Searching for: ${name}`);
```

Comment:
Remove any unused code to keep everything clean

Reply:
removed